src/test/java/io/confluent/connect/jdbc/source/integration/BaseOOMIntegrationTest.java

```diff
@@ -47,6 +47,7 @@ public void stopTask() {
   @Test
   public void testStreamingReads() throws InterruptedException {
```

Comment:
@Tanish0019  - can you pls add one test here for successful start if tables have not been fetched also...otherwise LGTM!